src/os/mixin/urltilemixin.js

```diff
@@ -414,6 +424,13 @@ ol.source.UrlTile.prototype.onImageLoadOrError_ = function(evt) {
     this.loadCount_++;
   } else {
     this.errorCount_++;
+
+    if (!this.authAlerted_) {
```

Comment:
It looks like `alertAuth` handles deduplication of alerts, but should `authAlerted_` also be set to true here to avoid trying to alert again?

Reply:
Yep, meant to take this out when I moved that to the auth module.